lua/tmux/navigate.lua

```diff
@@ -35,7 +56,7 @@ local function navigate_to(direction)
 	if is_border(direction) then
```

Comment:
for clarity we should refactor is_border into is_nvim_border. Thus, we can 

```lua
local function is_nvim_border(border)
	return winnr() == winnr("1" .. border)
end
[...]
if is_nvim_border(direction) and has_tmux_target(direction) then
    wrapper.change_pane(direction)
elseif is_nvim_border(direction) and cfg.options.navigation.cycle_navigation then
    wrap_window(direction)
else
    wincmd(direction)
end
```

This ensures the control flow is visible on the most upper scope.